src/app/reports-handler.ts

```diff
@@ -82,6 +83,11 @@ export const exportLogs = (): void => {
         });
         return;
     }
+
+    for (const log of logs) {
+        fs.writeFileSync(path.join( source, log.filename + '.log' ), log.contents );
```

Comment:
`ILogFile.filename` should contain the log file extension as well

Reply:
The existing code which packages log files only includes files with .log extension. So if we want to send the extension with the filename, should we then extract the extension of log files and add to the list of log files to be packaged? It is not clear to me if this might have other implications?

Reply:
I  might want to send .json files, .har, or whatever that might be useful logs

Reply:
But I assume there is a reason the "generateArchiveForDirectory" was restricted to only include .log files? If we allow for any extensions, it means any files that happen to be in the source folder will be included (not just those we send with retrieve logs), instead of just the .log files as we do today. Which might be what we want, I am just pointing out that it is a change in behaviour that I have no idea what impact it could have.  Also, you could still send a .json file, it will just get an extra ".log" extension when saved.

Reply:
Can't you just send any file in that folder that ends wit h`.log`, **and** any file returned in the new API?

Reply:
No, the API for creating the zip file works on a whole folder.